CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ find_package(GMock REQUIRED)
 find_package(GTest REQUIRED)
 find_package(Benchmark)
 
+SET(ENABLE_LIBAIO OFF)
```

Comment:
We don't need a variable controlled by user to indicate whether the LIBAIO is enabled. We could do it automatically. I mean, if we found libaio, build the codes related with libaio. If we couldn't find libaio, then just skip the codes depends libaio. We could refer to the implementation of `Benchmark`. We need to emit an macro too.